test/testcafe/framework/page-objects/EnrollWebauthnPageObject.js

```diff
@@ -13,6 +13,13 @@ export default class EnrollWebauthnPageObject extends BasePageObject {
     return this.form.elementExist('.idx-webauthn-enroll-text');
   }
 
+  hasWebauthnNotSupportedError() {
+    if (userVariables.gen3) {
+      return this.form.hasErrorBox();
```

Comment:
We may want to check a little deeper, like what text is there since this will return `true` as long as _any error_ appears.

Reply:
We expect that there is no any error on a page here